app/src/main/java/org/jellyfin/androidtv/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -432,6 +432,7 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item,
     private View.OnKeyListener keyListener = new View.OnKeyListener() {
         @Override
         public boolean onKey(View v, int keyCode, KeyEvent event) {
+            if (event.getAction() == KeyEvent.ACTION_DOWN) {
```

Comment:
The formatting inside this if statement is wrong. Please indent it or change the code to return false when the action is not ACTION_DOWN

Reply:
What are the current indent preferences for java files?

Reply:
It's 4 spaces currently

Reply:
Should be fixed with the most recent commit